.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
   - .travis/maybe-bump-version.sh
 after_success:
   - .travis/maybe-publish-coverage.sh
+  - .travis/maybe-publish-gh-pages.sh
```

Comment:
There is no `gh-pages` branch on this repository, do you really want to publish the dummy app? Is there a demo for something like this that's just blueprints?